cs/src/Management/TunnelManagementClient.cs

```diff
@@ -729,10 +732,11 @@ private string UserLimitsPath
         /// <remarks>
         /// Copied from Microsoft.VsSaaS.Common to avoid taking a dependency on that assembly.
         /// </remarks>
-        private class ErrorDetails
-        {
-            public string? Message { get; set; }
-            public string? StackTrace { get; set; }
+        private class ErrorDetails
+        {
+            public string? Message { get; set; }
+            public string? StackTrace { get; set; }
+            public string? detail { get; set; }

```

Comment:
Why `detail` starts with lowercase but `Message` and `StackTrace` are capitalized? Shouldn't `JsonSerializer` handle the lowercase translation (maybe with an additional attribute or serializer settings)?